index.html

```diff
@@ -4,44 +4,111 @@
+    <li class="item"><a href="index.html">Home</a></li>
+    <li class="item"><a href="about.html">About</a></li>
+   <li class="item has-submenu">
+    <a tabindex="0">Services <i class="fa-solid fa-angle-down"></i></a>
+        <ul class="submenu">
+            <li class="subitem"><a href="editing.html">Editing</a></li>
+            <li class="subitem"><a href="proofreading.html">Proofreading</a></li>
+            <li class="subitem"><a href="formatting.html">Manuscript Formatting</a></li>
+            <li class="subitem"><a href="tutoring.html">Tutoring</a></li>
+            <li class="subitem"><a href="curriculum.html">Curriculum Development</a></li>
+            <li class="subitem"><a href="research.html">Research Assistance</a></li>
+            
         </ul>
+    </li>
+    <li class="item"><a href="contact.html">Contact</a></li> 
+    <li class="toggle"><i class="fa-solid fa-bars"></i></li>
+</ul>
+</div>
     </nav>
 </header>    
 <main>
-    <div> 
-        <h1>Testimonials</h1>
-        <p>Testimonial from a client</p>
+   <div class="slideshow-container">
+
+    <div class="mySlides">
+        <q>First testimonial about services received.</q>
+        <p class="stars"><i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i></p>
+    </div>
+
+    <div class="mySlides">
+        <q>Second testimonial about services received.</q>
+        <p class="stars"><i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i></p>
     </div>
-   <div>
-       <img src="images/mss-logo.jpg" alt="Moore Student Success Logo">
+
+    <div class="mySlides">
+        <q>Third testimonial about services received.</q>
+        <p class="stars"><i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i>
+            <i class="fa-solid fa-star"></i></p>
+    </div>
+    <a class="prev" onclick="plusSlides(-1)"><i class="fa-solid fa-angle-left"></i></a>
+    <a class="next" onclick="plusSlides(1)"><i class="fa-solid fa-angle-right"></i></a>
+    
    </div>
-   <div>
-       <p>*Always free estimates on services*</p>
+   <div class="dot-containers">
+        <span class="dot" onclick="currentslide(1)"></span>
+        <span class="dot" onclick="currentslide(2)"></span>
+        <span class="dot" onclick="currentslide(3)"></span>
    </div>
-</main>
-<footer>
-    <div>
-        <h2>Copyright</h2>
-        <p>2022</p>
+   
+    <div class="estimate">
+        <p>*Always free estimates on services*</p>
     </div>
     <div>
-        <h2>Contact Us</h2>
-        <p>moorestudentsuccess@gmail.com</p>
+        <table class="indextable">
```

Comment:
tables should only be used as a data table / comparing data. use divs and flexbox for this layout

Reply:
The table is only a placeholder and I will be changing it this week. I wanted to try making a basic table to learn how. I never intended it to be permanent.

Reply:
table removed and updated with div boxes.